threedi_modelchecker/config.py

```diff
@@ -2391,6 +2392,63 @@ def is_none_or_empty(col):
             models.AggregationSettings.aggregation_method,
         ),
     ),
+    AllEqualCheck(
+        error_code=1152,
+        level=CheckLevel.WARNING,
+        column=models.AggregationSettings.timestep,
+    ),
+    QueryCheck(
+        error_code=1153,
+        level=CheckLevel.WARNING,
+        column=models.AggregationSettings.timestep,
+        invalid=Query(models.AggregationSettings).filter(
+            models.AggregationSettings.timestep
+            < Query(models.GlobalSetting.output_time_step)
+            .filter(first_setting_filter)
+            .scalar_subquery()
```

Comment:
I think you should do a join on global settings here somewhere; now you are comparing *all* aggregation settings (also the ones referring other global settings) against the output_time_step of the first global settingg.